engine/src/main/java/org/terasology/logic/players/PlayerFactory.java

```diff
@@ -99,30 +94,9 @@ public Vector3f findSpawnPositionFromLocationComponent(LocationComponent locatio
     }
 
     private float getHeightOf(ComponentContainer prefab) {
-        BoxShapeComponent box = prefab.getComponent(BoxShapeComponent.class);
-        if (box != null) {
-            return box.extents.getY();
-        }
-
-        CylinderShapeComponent cylinder = prefab.getComponent(CylinderShapeComponent.class);
-        if (cylinder != null) {
-            return cylinder.height;
-        }
-
-        CapsuleShapeComponent capsule = prefab.getComponent(CapsuleShapeComponent.class);
-        if (capsule != null) {
-            return capsule.height;
-        }
-
-        SphereShapeComponent sphere = prefab.getComponent(SphereShapeComponent.class);
-        if (sphere != null) {
-            return sphere.radius * 2.0f;
-        }
-
-        HullShapeComponent hull = prefab.getComponent(HullShapeComponent.class);
-        if (hull != null) {
-            AABB aabb = hull.sourceMesh.getAABB();
-            return aabb.maxY() - aabb.minY();
+        CharacterMovementComponent movementComponent = prefab.getComponent(CharacterMovementComponent.class);
+        if (movementComponent != null) {
+            return movementComponent.height;
         }
 
         logger.warn("entity {} does not have any known extent specification - using default", prefab);
```

Comment:
This can be a bit more specific now stating that the player entity does not have a `CharacterMovementComponent`.